package.json

```diff
@@ -28,12 +28,11 @@
     "rebuild-hard:dev": "npm run clean-slate && npm run build:dev",
     "rebuild-hard:prod": "npm run clean-slate && npm run build:prod",
     "precommit": "lint-staged",
-    "prepush": "lint-staged",
-    "prettier:base": "prettier --parser typescript --single-quote --trailing-comma es5 --tab-width 2 --no-semi --print-width 80 --write",
```

Comment:
Are you getting any errors or something that could help us debug that? I've run it on both my macbook and my PC without it. I can remove it from this PR but if we're gonna use prettier I feel like we should apply it to our SCSS files as well. I've got a branch lined up for that that I was gonna open after this.